src/webui/www/private/views/rss.html

```diff
@@ -0,0 +1,827 @@
+    <li class="separator"><a href="#rename"><img src="images/qbt-theme/edit-rename.svg" alt="QBT_TR(Rename...)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Rename...)QBT_TR[CONTEXT=RSSWidget]</a></li>
+    <li><a href="#delete"><img src="images/qbt-theme/edit-delete.svg" alt="QBT_TR(Delete)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Delete)QBT_TR[CONTEXT=RSSWidget]</a></li>
+
+    <li class="separator"><a href="#newSubscription"><img src="images/qbt-theme/document-new.svg" alt="QBT_TR(New subscription...)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(New subscription...)QBT_TR[CONTEXT=RSSWidget]</a></li>
+    <li><a href="#newFolder"><img src="images/qbt-theme/folder-new.svg" alt="QBT_TR(New folder...)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(New folder...)QBT_TR[CONTEXT=RSSWidget]</a></li>
+    <li class="separator"><a href="#updateAll"><img src="images/qbt-theme/view-refresh.svg" alt="QBT_TR(Update all feeds)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Update all feeds)QBT_TR[CONTEXT=RSSWidget]</a></li>
+
+    <li class="separator"><a href="#copyFeedURL" id="CopyFeedURL"><img src="images/qbt-theme/edit-copy.svg" alt="QBT_TR(Copy feed URL)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Copy feed URL)QBT_TR[CONTEXT=RSSWidget]</a></li>
+</ul>
+
+<ul id="rssArticleMenu" class="contextMenu">
+    <li><a href="#Download"><img src="images/qbt-theme/download.svg" alt="QBT_TR(Download torrent)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Download torrent)QBT_TR[CONTEXT=RSSWidget]</a></li>
+    <li><a href="#OpenNews"><img src="images/qbt-theme/application-x-mswinurl.svg" alt="QBT_TR(Open news URL)QBT_TR[CONTEXT=RSSWidget]" /> QBT_TR(Open news URL)QBT_TR[CONTEXT=RSSWidget]</a></li>
+</ul>
+
+<script>
+    'use strict';
+
+    if (window.qBittorrent === undefined) {
+        window.qBittorrent = {};
+    }
+
+    let serverSyncRssDataInterval = 1500;
+
+    window.qBittorrent.Rss = (function() {
+        const exports = function() {
+            return {
+                init: init,
+                unload: unload,
+                load: load,
+                addRSSFeed: addRSSFeed,
+                addFolder: addFolder,
+                showRssFeed: showRssFeed,
+                showDetails: showDetails,
+                updateRssFeedList: updateRssFeedList,
+                refreshFeed: refreshFeed,
+                refreshAllFeeds: refreshAllFeeds,
+                moveItem: moveItem,
+                removeItem: removeItem,
+                markItemAsRead: markItemAsRead,
+                markArticleAsRead: markArticleAsRead,
+                markSelectedAsRead: markSelectedAsRead,
+                openRssDownloader: openRssDownloader,
+                rssFeedTable: rssFeedTable
+            };
+        };
+
+        let feedData;
+        let lastFeedResponse;
+        let pathByFeedId;
+        let rssFeedContextMenu;
+        let rssArticleContextMenu;
+        let feedRefreshTimer;
```

Comment:
is it possible to give them a default value?

Reply:
The context menus didn't need to be in this scope so I moved them in to the init function. For `feedData` and `pathByFeedId` I provided a default. For `feedRefreshTimer` I don't think there is a good default.

While at it I also removed the exports that weren't being used.